app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -20,6 +20,15 @@ def call
           # Accept Date of Birth from both memorable date and input date components
           formatted_dob = MemorableDateComponent.extract_date_param flow_params&.[](:dob)
           flow_session[:pii_from_user][:dob] = formatted_dob if formatted_dob
+
+          if IdentityConfig.store.in_person_capture_secondary_id_enabled
+            if flow_session[:pii_from_user][:id_address_current] == 'true'
+              mark_step_complete(:residential_address)
+            end
+            mark_step_complete(:address)
+          else
+            mark_step_complete(:residential_address)
```

Comment:
This is a minor issue: the radio button text under the residential address question seems to wrap beyond the margins for the page
![Screen Shot 2023-03-02 at 9 33 14 AM](https://user-images.githubusercontent.com/102754632/222459163-bee5975a-eae9-4ebd-8972-9a41760c8382.png)

Reply:
Nice catch carmen!

Reply:
Thanks!  I'll fix this. 🙏🏻